sample/redo/lib/settings.dart

```diff
@@ -1,4 +1,4 @@
-final diffServerUrl = 'http://aa-mac.local:7001/pull';
+final diffServerUrl = 'http://10.0.2.2:7001/pull';
```

Comment:
Can you use the prod servers?

Reply:
Yes, it was just harder to test that way. Need to add some way to control from outside the code.